podcasts/smarthuette.yml

```
@@ -0,0 +1,15 @@
+name: "Smarthuette"
+website: https://podcast.smarthuette.de
+podcastIndexID: 7425625
+rssFeed: https://feeds.transistor.fm/smarthuette
+spotify: https://open.spotify.com/show/3cYGMZNGW9C8jVYEOx052k
+description: "Zwei Freunde, ihr Smart Home, Self-Hosting, Technik und das ein oder andere Interview mit Makern"
+tags:
+    - Smart Home
+    - Home Assistant
+    - Self-Hosting
+    - Linux
+    - Technology
+    - Maker
+    - Hardware
+    - Interviews
```

```suggestion
    - Interviews

```

Adding a line at the end of the file for yaml-lint